### DISCUSSION TITLE
Question: How to debug zigbee2mqtt failing to send messages (still receiving) until after reboot ...

### DISCUSSION THREAD

**Discussion by thiscantbeserious** [Koenkk/zigbee2mqtt#6358] [Devices]
I realize this is a difficult topic. Hence I didn't simply open up an issue since I have no clue where to start properly trying to debug it.

I'm using zigbee2mqtt 1.17.1 within homeassistant with slae.sh's CC2652RB coordinator (behind a main-powered USB-HUB) with the latest coordinator firmware (20210120) and a C2531 router (CC2531_router_2020_09_29) to extend the network. 

The network is not that big and all devices have an LQI above 50 and initially everything is responsive for hours, sometimes even days. These are the devices I have:

4x TS0601_thermostat
4x TT001ZAV20 sensors
2x ZL1000701-27-EU-V1A02 (those are currently not always on)

**Now every once in a while zigbee2mqtt fails to set the state of either a light or a thermostatic valve.** 

When that happens it usually happens to pull the whole network down with it in regards to setting any state.

The weird thing is that **it seems to still receive messages and able to read the LQI**. 

By "pulling down the network with it" I mean that any sent message will return **"No network route (205)"**. 

That first happens just for a single device, but soon enough all of them follow - yet the coordinator still receives messages. 

**Restarting zigbee2mqtt doesn't help, waiting for a few hours doesn't help either** - I have to reboot the host (RPi 4) and then it works flawlessly again until it occurs next.

If I'd have to a really wild guess I'd were to say that it could it be related to the coordinator (firmware?), since it's the only constant that I have when it happens and rebooting will always fix it. 

I already tried removing the router and placing in another spot, switched Zigbee Channels, made sure that there are no interferences in regard to USB3, Bluetooth or 2.4GHZ WiFi Channels overlapping, replaced the antennas ... nothing helps. 

I've also read that ZLL might introduce weird side-effects in some old issues - but shouldn't the coordinator be able settle them out especially after a few hours (more then 4) - especially since it also happens for devices not connected to the lights itself, when they're off ... and it also happens for devices absolutely next to the coordinator itself.

**Comment by kapabel** [comment]
Hello! Did you end up finding a solution for your 205 problem?

**Reply by thiscantbeserious** [reply]
> Hello! Did you end up finding a solution for your 205 problem?

Just skimping trough my unread dicussions - didnt get notified sorry for that!

As for an answer:

No not really ... since one Hardware Dongle actually died I had to redo the whole setup re-installing and setting up the Addon again. In the process I actually also replaced the actively powered USB-Hub since apparently the older one seemed to shorten out the circut back to the host (so much for calling that protected ...).

Every since its working again and I just did what I deemed the most usable:

Just dont Update once its working - until I move everything to VMs where I can snapshot it properly before that shit starts breaking down.

Apparently that route is working out good with Homeassistant and zigbee2mqtt for now

**Reply by kapabel** [reply]
Many thanks for your reply. I don't remember how I have solved my issue in the end. It was a combination of many things wrongly setup but now its working without knowing exactly what I did to make it work. Never stop a running system.
